galaxy_ng/app/api/ui/views/landing_page.py

```diff
@@ -0,0 +1,153 @@
+from rest_framework.permissions import IsAuthenticatedOrReadOnly
+from rest_framework.response import Response
+from pulp_ansible.app.models import Collection
+from galaxy_ng.app.models import Namespace
+
+from galaxy_ng.app.api import base as api_base
+
+
+class LandingPageView(api_base.APIView):
+    permission_classes = [IsAuthenticatedOrReadOnly]
+
+    def get(self, request, *args, **kwargs):
+        collection_count = Collection.objects.count()
```

Comment:
This needs more filtering, not all `Collections` should be counted, we want to show the count only on `published` collections.

This queryset needs to filter only the collections that are on `/published` repo. (we have a constant GOLDEN_REPO on constants file)

Reply:
@rochacbruno can you tell me which file it is? I cannot find anything repo related constant.

Reply:
And then the code would look like this?

```py
distro = AnsibleDistribution.objects.get(base_path=golden_name)
repository_version = distro.repository.latest_version()
collection_count = Collection.objects.filter(pk__in=repository_version.content).count()
```

Reply:
@brumik yes, sorry the proper one is that https://github.com/ansible/galaxy_ng/blob/master/galaxy_ng/app/settings.py#L37